src/index/sparse/sparse_inverted_index.h

```diff
@@ -200,6 +200,9 @@ class InvertedIndex : public BaseInvertedIndex<T> {
         for (size_t i = 0; i < rows; ++i) {
             amount += data[i].size();
         }
+        if (amount == 0) {
+            return Status::success;
```

Comment:
this early exit does not set rhe following values (lines 222-223)
```
        value_threshold_ = *pos;
        drop_during_build_ = true;
```
is it expected?

Reply:
yes, they have default values of 0 and false.